tests/integration/scenarios/dash_live_mutli_periods.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from "chai";
```

Comment:
Look at the filename.

Also that's not

Reply:
I don't see the prolbem ...